tabUtil/tabhttp.py

```diff
@@ -131,11 +133,25 @@ def export_view(configs, view, format, logger):
         protocol = u'https'
     else:
         protocol = u'http'
+        
```

Comment:
Interestingly, whatever value is provided for size first is used, but for format, the last value is used. So:

myworkbook/myview/?:size=50,50&:format=png&:size=100,100&:format=csv

...returns a CSV file. But:

myworkbook/myview/?:size=50,50&:format=png&:size=100,100

...returns a 50x50 PNG. Which is actually pretty safe behavior for the way VizAlerts would currently work.

So while I'd like to lock down which could be used and which would be ignored by VizAlerts, I think we're pretty safe with the current code, as an initial release.